.browserslistrc

```diff
@@ -6,7 +6,6 @@ not dead
 Chrome >= 45
 Firefox >= 38
 Edge >= 12
-Explorer >= 10
```

Comment:
IE11 will be supported by MS in some way under Oct 14th 2025. Consider changing to 
`Explorer > 10`

Reply:
Not really: https://techcommunity.microsoft.com/t5/Windows-IT-Pro-Blog/The-perils-of-using-Internet-Explorer-as-your-default-browser/ba-p/331732